docs/getting-started/sw-lib.md

```diff
@@ -0,0 +1,195 @@
+---
+layout: index
+title: Using sw-lib
+navigation_weight: 2
+---
+
+# Using sw-lib
+
+The `sw-lib` module is a high-level library that lets you
+easily configure a service worker for precaching assets
+during install as well as a configuring a number of
+runtime caching strategies.
+
+
+## Install
+
+Install the module with NPM.
+
+```
+npm install --save-dev sw-lib
+```
+
+You'll need to copy the following files from
+`./node_modules/sw-lib` into your project.
+
+* sw-lib.js
+* sw-lib.js.map
+* sw-lib.min.js
+* sw-lib.min.js.map
+
+You can do this with a simple command like:
+
+```
+cp node_modules/sw-lib/build/* app/third-party/sw-lib/
```

Comment:
Matt,

In my experiments, I originally copied the files to a subdirectory but found that some functionality didn't work. I think (if I remember correctly) that precaching worked but runtime didn't. That's when I slapped my head and said, "Service workers, duh!." I moved it to the root and everything was fine.

Is there something I overlooked?

@jeffposnick